### PR TITLE
fix(kit): `Input` fix mask

### LINKED ISSUE #3094: 🐞 - Input mask doesn't work on DEMO
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/components/input#mask

### Is this issue blocking you?

Non-Blocking

### Description

image

+ add e2e

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #3094

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/cdk/schematics/ng-update/steps/migrate-templates.ts [hunk 1]
```diff
@@ -376,15 +376,11 @@ function migrateBinaryAttributes({
...
-        const elements = findElementsInTemplateByFn(
-            template,
-            el =>
-                el.tagName !== `tui-hosted-dropdown` &&
-                el.attrs?.some(
-                    attr =>
-                        attr.value === 'true' &&
-                        attr.name.includes(attrName.toLowerCase()),
-                ),
+        const elements = findElementsInTemplateByFn(template, el =>
```
Comment (splincode) [inline projects/cdk/schematics/ng-update/steps/migrate-templates.ts:379]: image

**Review by splincode — APPROVED** [review]

File: projects/demo-integrations/cypress/tests/desktop/kit/input/input.cy.ts [hunk 2]
```diff
@@ -112,4 +111,13 @@ describe(`Input`, () => {
         cy.get(`@input`).scrollTo(`center`, {ensureScrollable: false});
         cy.get(`@wrapper`).matchImageSnapshot(`08-horizontally-scrolled`);
     });
+
+    it(`external mask works`, () => {
+        cy.tuiVisit(`${INPUT_PAGE_URL}`);
+
+        cy.get(`tui-doc-example[id=mask]`).tuiScrollIntoView().as(`wrapper`);
+        cy.get(`@example`).find(`tui-input input[tuiTextfield]`).as(`input`);
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/desktop/kit/input/input.cy.ts:119]: wrapper

Comment (splincode) [reply projects/demo-integrations/cypress/tests/desktop/kit/input/input.cy.ts:119]: image

Comment (waterplea) [reply projects/demo-integrations/cypress/tests/desktop/kit/input/input.cy.ts:119]: Someday I'll get it right :D

**Review by splincode — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**